proposals/0426-bitwise-copyable.md

```diff
@@ -283,7 +283,7 @@ public func storeBytes<T : BitwiseCopyable>(
 
 This allows for optimal code generation because `memcpy` instead of value witnesses can be used.
 
-Additionally, we propose deprecating the original, unconstrained overloads of `loadUnaligned` and `storeBytes`.
+The pre-existing methods will continue to exist (see [alternatives considered](#deprecation).)
```

Comment:
```suggestion
The existing methods that use a runtime assert instead of a type constraint will still be available (see [alternatives considered](#deprecation)).
```